lib/rubocop/cop/mixin/frozen_string_literal.rb

```diff
@@ -34,7 +34,9 @@ def frozen_string_literals_enabled?
       end
 
       def leading_comment_lines
-        processed_source[0..2].compact
+        processed_source.comments.select do |comment|
```

Comment:
We could use an `each_with_object` here to avoid chaining `map` onto `end`.

```ruby
processed_source.comments.each_with_object([]) do |comment, collection|
  next if comment.loc.line > 3
  collection << comment.text
end
```

Reply:
Thanks for the review. I arranged it slightly and applied it.